### PR TITLE
OCM-7552 | feat: output IAM policies that are being attached to IAM roles

### PR DESCRIPTION
**Pull request by marcolan018**
https://issues.redhat.com/browse/OCM-7552

samples:
output will include policy arn when attach non-awsmanagedpolicy during account-roles creation
```
I: Creating classic account roles using 'arn:aws:iam::765374464689:user/llan@redhat.com'
I: Created role 'marco-ControlPlane-Role' with ARN 'arn:aws:iam::765374464689:role/marco-ControlPlane-Role'
I: Attached policy 'arn:aws:iam::765374464689:policy/marco-ControlPlane-Role-Policy' to role 'marco-ControlPlane-Role'

I: Created role 'marco-Worker-Role' with ARN 'arn:aws:iam::765374464689:role/marco-Worker-Role'
I: Attached policy 'arn:aws:iam::765374464689:policy/marco-Worker-Role-Policy' to role 'marco-Worker-Role'

I: Created role 'marco-Support-Role' with ARN 'arn:aws:iam::765374464689:role/marco-Support-Role'
I: Attached policy 'arn:aws:iam::765374464689:policy/marco-Support-Role-Policy' to role 'marco-Support-Role'

I: Created role 'marco-Installer-Role' with ARN 'arn:aws:iam::765374464689:role/marco-Installer-Role'
I: Attached policy 'arn:aws:iam::765374464689:policy/marco-Installer-Role-Policy' to role 'marco-Installer-Role'
```

output will include policy link when attach non-awsmanagedpolicy during operator-roles creation
```
I: Creating roles using 'arn:aws:iam::765374464689:user/llan@redhat.com'
I: Created role 'marco-test-z4a6-kube-system-kms-provider' with ARN 'arn:aws:iam::765374464689:role/marco-test-z4a6-kube-system-kms-provider'
I: Attached policy 'ROSAKMSProviderPolicy(https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ROSAKMSProviderPolicy)' to role 'marco-test-z4a6-kube-system-kms-provider'

I: Created role 'marco-test-z4a6-openshift-image-registry-installer-cloud-credent' with ARN 'arn:aws:iam::765374464689:role/marco-test-z4a6-openshift-image-registry-installer-cloud-credent'
I: Attached policy 'ROSAImageRegistryOperatorPolicy(https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ROSAImageRegistryOperatorPolicy)' to role 'marco-test-z4a6-openshift-image-registry-installer-cloud-credent'

I: Created role 'marco-test-z4a6-openshift-ingress-operator-cloud-credentials' with ARN 'arn:aws:iam::765374464689:role/marco-test-z4a6-openshift-ingress-operator-cloud-credentials'
I: Attached policy 'ROSAIngressOperatorPolicy(https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ROSAIngressOperatorPolicy)' to role 'marco-test-z4a6-openshift-ingress-operator-cloud-credentials'

I: Created role 'marco-test-z4a6-openshift-cluster-csi-drivers-ebs-cloud-credenti' with ARN 'arn:aws:iam::765374464689:role/marco-test-z4a6-openshift-cluster-csi-drivers-ebs-cloud-credenti'
I: Attached policy 'ROSAAmazonEBSCSIDriverOperatorPolicy(https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ROSAAmazonEBSCSIDriverOperatorPolicy)' to role 'marco-test-z4a6-openshift-cluster-csi-drivers-ebs-cloud-credenti'

I: Created role 'marco-test-z4a6-openshift-cloud-network-config-controller-cloud-' with ARN 'arn:aws:iam::765374464689:role/marco-test-z4a6-openshift-cloud-network-config-controller-cloud-'
I: Attached policy 'ROSACloudNetworkConfigOperatorPolicy(https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ROSACloudNetworkConfigOperatorPolicy)' to role 'marco-test-z4a6-openshift-cloud-network-config-controller-cloud-'

I: Created role 'marco-test-z4a6-kube-system-kube-controller-manager' with ARN 'arn:aws:iam::765374464689:role/marco-test-z4a6-kube-system-kube-controller-manager'
I: Attached policy 'ROSAKubeControllerPolicy(https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ROSAKubeControllerPolicy)' to role 'marco-test-z4a6-kube-system-kube-controller-manager'

I: Created role 'marco-test-z4a6-kube-system-capa-controller-manager' with ARN 'arn:aws:iam::765374464689:role/marco-test-z4a6-kube-system-capa-controller-manager'
I: Attached policy 'ROSANodePoolManagementPolicy(https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ROSANodePoolManagementPolicy)' to role 'marco-test-z4a6-kube-system-capa-controller-manager'

I: Created role 'marco-test-z4a6-kube-system-control-plane-operator' with ARN 'arn:aws:iam::765374464689:role/marco-test-z4a6-kube-system-control-plane-operator'
I: Attached policy 'ROSAControlPlaneOperatorPolicy(https://docs.aws.amazon.com/aws-managed-policy/latest/reference/ROSAControlPlaneOperatorPolicy)' to role 'marco-test-z4a6-kube-system-control-plane-operator'
```

### CODE REVIEW

File: pkg/aws/policies.go [hunk 1]
```diff
@@ -35,9 +35,18 @@ import (
 	client "github.com/openshift/rosa/pkg/aws/api_interface"
 	"github.com/openshift/rosa/pkg/aws/tags"
 	"github.com/openshift/rosa/pkg/helper"
+	"github.com/openshift/rosa/pkg/reporter"
 )
 
-var DefaultPrefix = "ManagedOpenShift"
+const (
+	awsManagedPolicyRegexPattern = `^arn:aws:iam::aws:policy/.*$`
+	awsManagedPolicyUrlPrefix    = "https://docs.aws.amazon.com/aws-managed-policy/latest/reference/"
```
Comment (gdbranco) [inline pkg/aws/policies.go:43]: I wonder how problematic might this be, considering that if AWS changes this url we'll need a new release :thinking:

Comment (marcolan018) [reply pkg/aws/policies.go:43]: talked with @gdbranco , one option to fix this is to make the URL prefix tweakable in CS side, and CLI can read it from the API when listing the policies. Maybe we can track this in a new issue.
@ciaranRoche wdyt?

Comment (ciaranRoche) [reply pkg/aws/policies.go:43]: Would it be worth create a ticket for this, and following up? Use the ticket to get more context as to how often or how likely it is the AWS will change the URL? And if they do would it have a redirect? 

If the juice is worth the squeeze we can make the API changes. But right now I think we are good with this until we get Aaren to get us to rest of the context.

Comment (marcolan018) [reply pkg/aws/policies.go:43]: https://issues.redhat.com/browse/OCM-10351
issue created to track this

**Review by ciaranRoche — COMMENTED** [review]
/lgtm

**Pull request merged**